Assets/MixedRealityToolkit/MSBuild/Scripts/CSProjectInfo.cs

```diff
@@ -0,0 +1,363 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if UNITY_EDITOR
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.Compilation;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.MSBuild
+{
+    /// <summary>
+    /// From which source was the project created.
+    /// </summary>
+    public enum ProjectType
+    {
+        /// <summary>
+        /// The project is backed by an Assembly-Definition file.
```

Comment:
>Assembly-Definition [](start = 40, length = 19)

Nit: assembly definition (capitilization here and down below) or make every usage of this term have this capitalization (just to be consistent at least within this file)